__tests__/compiler/watcher_proccess.js

```diff
@@ -0,0 +1,6 @@
+require("ts-node/register/transpile-only");
```

Comment:
Before all tests were compiled to .js as a separate step, so child process could run `watcher_proccess.js`. Now all tests are compiled in-memory with `ts-jest` (for better watching DX and potential for more optimization in the future), but that doesn't work with child processes. Though maybe it would be better to load `ts-node/register/transpile-only` as a part of fork call.